frontends/web/src/api/accountsync.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright 2021 Shift Crypto AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { subscribeEndpoint } from './subscribe';
+import { Unsubscribe } from '../utils/event';
+import { getAccounts, IAccount} from './account';
+
+export const accounts = (cb: (acconuts: IAccount[]) => void): Unsubscribe => {
```

Comment:
typo on "acconuts"... did you actually test this somehow? :)

Reply:
also, would appreciate doc comments. does this function calls `cb` with a list of all accounts whenever then change? just guessing, but I wish I didn't have to.

Reply:
acconuts 🙈  yes and it works, the variable name is wrong but just used to pass to the cb

>  just guessing

correct with reading 👍  , I'll to add comments

Reply:
and yes I tested api/accounts , but the other PR api/devices I only looked at the backend